danielsaidi/RichTextKit
# [BUGFIX] Fix alignment 💯

# What this PR do:
- Fixes text alignment setting
# How to test it
- Run app and see for yourself.


https://github.com/danielsaidi/RichTextKit/assets/17381941/60f21e75-26f8-440e-b73f-38523ba7b268
